mutable_tree.go

```diff
@@ -365,14 +376,41 @@ func (tree *MutableTree) DeleteVersion(version int64) error {
 		return cmn.ErrorWrap(ErrVersionDoesNotExist, "")
 	}
 
-	tree.ndb.DeleteVersion(version)
+	tree.ndb.DeleteVersion(version, true)
 	tree.ndb.Commit()
 
 	delete(tree.versions, version)
 
 	return nil
 }
 
+// deleteVersionsFrom deletes tree version from disk specified version to latest version. The version can then no
+// longer be accessed.
+func (tree *MutableTree) deleteVersionsFrom(version int64) error {
+	newLatestVersion := version - 1
+	lastestVersion := tree.ndb.getLatestVersion()
+	for {
+		if version > lastestVersion {
+			break
+		}
+		if version == 0 {
```

Comment:
What about `version < 0`

Reply:
Also, this check is independent of the for loop (and hence, should live outside of the loop)

Reply:
Actually, version won't less than zero. If version is less than zero, an error will be return in an early [position](https://github.com/tendermint/iavl/blob/e0b4af63cf880b22f68fb03c5495a14eb4501654/mutable_tree.go#L247). But changing `version == 0` to `version <= 0` is much better.